voidcrew/modules/cloning/code/objects_datums.dm

```diff
@@ -0,0 +1,61 @@
+//objects
+
+/obj/effect/countdown/clonepod
+	name = "cloning pod countdown"
+	color = "#18d100"
+	text_size = 1
+
+/obj/effect/countdown/clonepod/get_value()
+	var/obj/machinery/clonepod/C = attached_to
+	if(!istype(C))
+		return
+	else if(C.occupant)
+		var/completion = round(C.get_completion())
+		return completion
+
+//datums
+
+/datum/brain_trauma
+	var/clonable = TRUE // will this transfer if the brain is cloned?
+
+/datum/brain_trauma/proc/on_clone()
+	if(clonable)
+		return new type
+
+/datum/brain_trauma/mild/phobia/on_clone()
+	if(clonable)
+		return new type(phobia_type)
+
+/datum/brain_trauma/special/psychotic_brawling/bath_salts
+	clonable = FALSE
+
+/datum/brain_trauma/special/beepsky
+	clonable = FALSE
+
+/datum/mood_event/cloned_corpse
+	description = "<span class='boldwarning'>I recently saw my own corpse...</span>\n"
+	mood_change = -6
+
+/datum/quirk/proc/clone_data() //return additional data that should be remembered by cloning
+/datum/quirk/proc/on_clone(data) //create the quirk from clone data
+
+//techweb
+/datum/techweb_node/cloning
```

Comment:
Please move this to the other techweb nodes as shown in https://github.com/Willardstation/tg-voidcrew/pull/79

Reply:
will do